src/releng_notification_identity/releng_notification_identity/config.py

```diff
@@ -0,0 +1,8 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from __future__ import absolute_import
+
+
+PROJECT_NAME = 'releng_notification_identity'
```

Comment:
project name should be ``releng-notification-identity`` and you then need to create ``PROJECT_PATH_NAME`` as in releng_notification_policy